server/data.js

```diff
@@ -0,0 +1,105 @@
+const _ = require('underscore');
+
+// 'YYYY-MM-DD HH:mm:ss' for MySQL insertion (extra quotes yes, 0 padding not needed)
+const format = (dateTime) => {
+  const y = dateTime.getFullYear();
+  const mon = dateTime.getMonth() + 1;
+  const d = dateTime.getDate();
+  const h = dateTime.getHours();
+  const min = dateTime.getMinutes();
+  const s = dateTime.getSeconds();
+  return `"${y}-${mon}-${d} ${h}:${min}:${s}"`;
+};
+
+/*
+ *  Generate data for the "inventory" table
+ */
+
+const partyTypes = {
+  // all: _.range(1, 21),
```

Comment:
maybe a more descriptive variable name for this function could help me understand its logics.

Reply:
also, judging from the commented out property of all: _.range(1,21), is small supposed be range (1, 19)?

Reply:
unclear how to improve clarity of the object name... it describes small parties, large parties, etc.  also, small parties are defined as parties of 1-3.  Parties of 19 would be large.  leave the names as-is.